src/components/ViewContainer/index.tsx

```diff
@@ -5,15 +5,14 @@ import makeStyles from '@material-ui/styles/makeStyles';
 const useStyles = makeStyles((theme: Theme) => ({
   root: {
     paddingTop: theme.spacing(2),
-    paddingBottom: theme.spacing(11),
```

Comment:
I added this `padding` to avoid situations like below.

![image](https://user-images.githubusercontent.com/61630877/119048399-76ebf780-b9bf-11eb-9c61-a7248e7cfd0a.png)

Reply:
Ok, I take a look if my changes negatively affect on it

Reply:
Indeed, it causes a problem, so I brought back `padding bottom`